applications/callflow/src/module/cf_log_in_out_ring_group.erl

```diff
@@ -0,0 +1,91 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%% Logs a user in or out of a ring group
+%%% @end
+%%% @contributors
+%%%   Max Lay
+%%%-------------------------------------------------------------------
+-module(cf_log_in_out_ring_group).
```

Comment:
Naming is hard! `"module":"log_in_out_ring_group"` just seems...verbose. What about `cf_ring_group_toggle`?